### DISCUSSION TITLE
Are there any plans to allow for more than 1024 redirects in vercel.json or allow .htaccess files in Vercel (or next.config.js)?

### DISCUSSION THREAD

**Discussion by felixmeziere** [vercel/vercel#5908] [Ideas] [answered]
Are there any plans to allow for more than 1024 redirects in vercel.json or allow .htaccess files in Vercel (or next.config.js)?

My company heavily needs this feature as we are very SEO-dependent and have thousands of product pages whose slug have been changing and will change a few times in the next months.

Cheers :-)

**Comment by NWRichmond** [comment, marked as answer]
Hey @felixmeziere, just wanted to pass along this relevant article that I found helpful:

>  _[How can I increase the limit of redirects or use dynamic redirects on Vercel?](https://vercel.com/support/articles/how-can-i-increase-the-limit-of-redirects-or-use-dynamic-redirects-on-vercel)._

**Reply by felixmeziere** [reply]
Thanks a lot, did use dynamic redirects from within next js in the end!
